docs/macvlan.md

```diff
@@ -285,7 +299,64 @@ $ ip -6 route
   2001:db8:abc4::/64 dev eth0  proto kernel  metric 256
   2001:db8:abc8::/64 dev eth0  proto kernel  metric 256
   default via 2001:db8:abc8::10 dev eth0  metric 1024
+  
+# Containers can have both v4 and v6 addresses assigned on a single container
```

Comment:
Can we rephrase to
`Containers can have both v4 and v6 addresses assigned to their interfaces` or
`Both v4 and v6 addresses can be assigned to the container's interface`